test/hull_test.cpp

```diff
@@ -94,3 +94,54 @@ TEST(Hull, Sphere) {
   EXPECT_FLOAT_EQ(sphereHull.GetProperties().volume,
                   sphere.GetProperties().volume);
 }
+
+// TEST(Hull, FailingTest1) {
```

Comment:
Instead of commenting these tests, you can change them to `TEST(Hull, DISABLED_FailingTest1)` - this way they get compiled, but just not run, and we get a warning to remind us.